server/routes/user.js

```diff
@@ -176,5 +201,65 @@ router.get('/projxTeam', function(req, res, next) {
 });
 
 
+/**
```

Comment:
i think this route could be replaced with `/getAdmin` and only return admin, since the frontend shouldn't have to know about all the users. sending over all the users might also be a lot of data